Documentation/InputSimulation/InputSimulationService.md

```diff
@@ -76,51 +77,57 @@ Camera position and rotation angles can be set explicitly in the tools window, a
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/Z7L4I1ET7GU" class="center" frameborder="0" allow="accelerometer; encrypted-media; gyroscope; picture-in-picture" allowfullscreen />
 
-## Hand simulation
+## Controller simulation
 
-The input simulation supports emulated hand devices. These virtual hands can interact with any object that supports regular hand devices, such as buttons or grabbable objects.
+The input simulation supports emulated controller devices (i.e. motion controllers and hands). These virtual controllers can interact with any object that supports regular controllers, such as buttons or grabbable objects.
 
-### Hand simulation mode
+### Controller simulation mode
 
-In the [input simulation tools window](#input-simulation-tools-window) the **Hand Simulation Mode** setting switches between two distinct input models. The default mode can also be set in the input simulation profile.
+In the [input simulation tools window](#input-simulation-tools-window) the **Default Controller Simulation Mode** setting switches between three distinct input models. This default mode can also be set in the input simulation profile.
 
 * *Articulated Hands*: Simulates a fully articulated hand device with joint position data.
```

Comment:
Any reason we're changing the input simulation language from hands to controllers? Seeing Controller simulation mode to me feels like the main focus is on simulating controller inputs (ie, the joystick, 6dof movement, triggers and buttons) rather than what the hands currently do, which is focused on pinching, poking, grasping objects in the world. Calling the articulated hands a controller device feels kind of off to me, as a more general term like input device makes more sense to me as a user reading the documentation.